source/Main.cpp

```diff
@@ -642,13 +667,13 @@ int APIENTRY wWinMain(_In_ HINSTANCE hInstance,	_In_opt_ HINSTANCE hPrevInstance
 				//List of files
 				ImGui::Text("Select filename");
 
-				ImGui::ListBoxHeader("", ImVec2(290, 110));
+				ImGui::ListBoxHeader("##", ImVec2(290, 110));
 
 				for (int i = 0; i < numFilenames; ++i)
 				{
 					if (ImGui::Selectable(filenamesPartial[i], &selectedFilenames[i]))
 					{
-						ZeroMemory(selectedFilenames, sizeof(selectedFilenames));
+						memset(selectedFilenames, 0, sizeof(selectedFilenames));

```

Comment:
OH, It's so painful 😢 
If we'll change `selectedFilenames`'s type to std::array, there'll fill method for this stuff

Reply:
Just fast fix 🥲